Tests/test_PDB.py

```diff
@@ -679,9 +679,11 @@ def test_comparison_entities(self):
         self.assertGreaterEqual(struct2, struct)
 
         # Model
-        self.assertTrue(model == model)  # __eq__ same type
+
+        self.assertEqual(model, model)  # __eq__ same type
         self.assertNotEqual(struct[0], struct[1])
 
+
```

Comment:
Tiny style issue here reported via flake8,

```
$ flake8
./Tests/test_PDB.py:687:9: E303 too many blank lines (2)
```

You added a blank line which is didn't like.